### PR TITLE
Test reading file

### PR DESCRIPTION
**Pull request by Filbird**
For now simply reads a file specified by the emitterPath and prints line by line to stdout.

### CODE REVIEW

File: logservice.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	if err := scanner.Err(); err != nil {
+		return fmt.Errorf("Unexpected scan error: %v", err)
+	}
+
+	return nil
+}
+
+func callScan(file, buildID string) {
+	scanFile(os.Stdout, file, buildID)
+	os.Exit(0)
```
Comment (stjohnjohnson) [inline logservice.go]: Why are you doing this?

File: logservice.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+func main() {
+	app := cli.NewApp()
+	app.Name = "logservice"
+	app.Usage = "For now, scans a file and prints to stdout"
+	app.UsageText = "logservice [options] build-id"
+	app.Copyright = "(c) 2016 Yahoo Inc."
+
+	if VERSION == "" {
```
Comment (d2lam) [inline logservice.go]: When does `VERSION` get set?

Comment (d2lam) [reply logservice.go]: If this is the Environment Variable set by incrementVersion, don't you need to get it by `os.Getenv("VERSION")`

File: logservice.go [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/urfave/cli"
+)
+
+var VERSION string
```
Comment (jer) [inline logservice.go]: To use this, we need to use LDFLAGS like this: https://github.com/screwdriver-cd/gitversion/blob/master/wercker.yml#L31

File: logservice.go [hunk 4]
```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/urfave/cli"
```
Comment (jer) [inline logservice.go]: I feel like this brings in more complexity than is required for this program

File: logservice.go [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+		return fmt.Errorf("Unexpected error opening the file: %v", err)
+	}
+
+	scanner := bufio.NewScanner(body)
+
+	for scanner.Scan() {
+		fmt.Fprintln(output, scanner.Text())
+	}
+	if err := scanner.Err(); err != nil {
+		return fmt.Errorf("Unexpected scan error: %v", err)
```
Comment (jer) [inline logservice.go]: This isn't a very useful addition to the error message that the scanner will give you. Either just return the error or add some context. The same for the error above.

File: logservice.go [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+func scanFile(output io.Writer, file, buildID string) error {
+	body, err := open(file)
+	if err != nil {
+		return fmt.Errorf("Unexpected error opening the file: %v", err)
+	}
+
+	scanner := bufio.NewScanner(body)
+
+	for scanner.Scan() {
```
Comment (jer) [inline logservice.go]: You can do this with io.Copy: https://godoc.org/io#Copy

File: README.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
+# Screwdriver Log Service
+[![Build Status][wercker-image]][wercker-url] [![Open Issues][issues-image]][issues-url]
+[![Go Report Card][goreport-image]][goreport-url]
+
+> ""
```
Comment (stjohnjohnson) [inline README.md:5]: What is this?

**Comment by jer** [comment]
👍

**Pull request merged**